.ci/ci-fast-return.sh

```diff
@@ -374,6 +470,121 @@ doc.md" "$matches"
 
 }
 
+testIsLabelSet() {
+	local result=""
+
+	result=$(is_label_set "" "" "")
+	assertEquals "0" "$result"
+
+	result=$(is_label_set "repo" "" "")
+	assertEquals "0" "$result"
+
+	result=$(is_label_set "repo" "pr" "")
+	assertEquals "0" "$result"
+
+	result=$(is_label_set "" "pr" "label")
+	assertEquals "0" "$result"
+
+	result=$(is_label_set "repo" "" "label")
+	assertEquals "0" "$result"
```

Comment:
It wouldn't hurt to have a negative test that would ensure 1 is returned when expected.

Reply:
Agreed. I've added some logic to `is_label_set()` (see `testing_mode=`) to handle this scenario by faking the return value from that function (after having checked the params).